gen2-play-encoded-stream/main.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import depthai as dai
+import subprocess as sp
+from os import name as osName
+
+# Create pipeline
+pipeline = dai.Pipeline()
+
+# Define sources and output
+camRgb = pipeline.createColorCamera()
+videoEnc = pipeline.createVideoEncoder()
+xout = pipeline.createXLinkOut()
+
+xout.setStreamName("h265")
+
+# Properties
+camRgb.setBoardSocket(dai.CameraBoardSocket.RGB)
+camRgb.setResolution(dai.ColorCameraProperties.SensorResolution.THE_4_K)
+videoEnc.setDefaultProfilePreset(3840, 2160, 30, dai.VideoEncoderProperties.Profile.H265_MAIN)
+
+# Linking
+camRgb.video.link(videoEnc.input)
+videoEnc.bitstream.link(xout.input)
+
+width, height = 720, 500
```

Comment:
To what width/height is this related?
(Check camRgb.getVideoSize / getVideoWidth/Height as well as for setDefaultProfilePreset)

Reply:
This is used only for the ffplay command (-x and -y arguments).

Reply:
This being the final displayed window size then?

Reply:
Yes, this is used just to lower the ffplay resolution, otherwise it opens a 3840x2160 window.